Dockerfile

```diff
@@ -10,4 +10,5 @@ COPY . .
 RUN npm install
 RUN npm install dotenv --save
 RUN npm test
-CMD ["npm", "start"]
+ENTRYPOINT ["npm","run"]
+#CMD ["npm", "start"]
```

Comment:
@nkumar-topcoder just curious what is purpose of using `ENTRYPOINT` instead of `CMD`?